### DISCUSSION TITLE
Image Doesn't show in Notification React Native Firebase

### DISCUSSION THREAD

**Discussion by MohammadAlderson** [invertase/react-native-firebase#6099] [Q&A] [answered]
I have a problem with images in notification with rnfirebase v5.6.0. I receive image from messaging and the image link correct but can't show the image in notification. It does not happen at all the time, it happens in some unknown situations. This is my display notification code
```
const notification = new firebase.notifications.Notification({
 sound: 'default',
 show_in_foreground: true,
})
 .setNotificationId(notificationId)
 .setTitle(notificationData.title)
 .setBody(notificationData.body)
 .setData(notificationData.data)
 .android.setColor('#ffffff')
 .android.setAutoCancel(true)
 .android.setChannelId('channelId')
 .android.setSmallIcon('ic_stat_name')
 .android.setPriority(firebase.notifications.Android.Priority.Max)
 .android.setBigPicture(message._android._bigPicture.picture)
 .android.setLargeIcon(message._android._largeIcon);

firebase.notifications().displayNotification(notification);
```
**For more information my react native version is 0.61.5 and my react-native-firebase version is 5.6.0.**

Note: image size is ok

**Comment by mikehardy** [comment, marked as answer]
With apologies, no effort will be spent examining anything to do with react-native-firebase v5, please update to current, noting that you will probably want to use a real notifications package like notifee.app in combination with modern react-native versions and modern react-native-firebase versions

All of these versions are old enough they are unsupportable

**Reply by MohammadAlderson** [reply]
Dear mike
I did as you said and I'm currently using notifee package and works fine
Thanks
